Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -6,7 +6,7 @@ for cloud native workloads. It is built on top of Cilium and eBPF to enable
 deep visibility into the communication and behavior of services as well as the
 networking infrastructure in a completely transparent manner.
 
-* Hubble can be configured to be in **local mode** or **distributed mode (beta)**.
+* Hubble can be configured to be in **local mode** or **distributed mode**.
```

Comment:
how about swapping these two, and show the `Distributed Mode` tab by default?

Reply:
Do we still need local mode?

Reply:
that's a good question, we could possibly remove the local mode that uses unix domain socket. you can still do:
```
--set global.hubble.listenAddress="localhost:4244"
```
if you want to avoid exposing the hubble API endpoint and only use `hubble` cli locally (with some overhead from going over TCP instead of unix domain socket i guess). @Rolinh @kAworu what do you think?

Reply:
@joestringer My guess is that local mode still has a couple of use cases, like single node cluster or to avoid an open port on the host network namespace.

@michi-covalent I'm not sure I understand the goal of listening on a localhost TCP port instead of the UNIX domain socket. In any case, the key difference here between TCP vs UNIX domain socket is that mTLS is enabled by default on TCP, and the hubble CLI does not yet support mTSL.

Reply:
@kAworu ah that makes sense, i misread the documentation, i thought enabling mTLS was optional for TCP.

personally i'm ok with keeping the "local mode" section.

Reply:
> Do we still need local mode?

Aside from what @kAworu mentioned already, we have the use-case for local mode as a replacement for cilium monitor (#12416). The local unix domain socket is also mounted on the Hubble Relay pod for peers discovery. I think the local mode is definitely useful.

Reply:
@michi-covalent I swapped the tabs, thanks for the suggestion.

Reply:
I was thinking less about whether there is a use case for local hubble access and more about whether we need to tell users to configure in either local or remote way. As far as I understand, users currently have to (1) Choose which mode they want to support and configure Cilium that way and (2) observe the cluster. I'm wondering how we can move towards (1) being automatic and then step (2) has the user decide whether they want local or cluster insights and just run the command in the right place to observe what's going on.

Reply:
one thing i was thinking about, maybe we should set `global.hubble.enabled` to `true` by default and get rid of the "local mode" tab altogether. this way the "local mode" is enabled by default, and you can run `hubble observe` from cilium pod.

it's not fully automatic since you still need to decide whether you want to enable distribute mode, but one step closer to what @joestringer is describing i think.

Reply:
let's address this in a separate PR. i filed https://github.com/cilium/cilium/issues/13311 for enabling hubble by default cc @kAworu @Rolinh

Reply:
@michi-covalent I think this makes sense, especially considering that we want to deprecate cilium monitor. Let's do this in a separate PR then.